### PR TITLE
Persons in bavet122

### PR DESCRIPTION
**Pull request by MassimoVilla**
Owner and scribe of MS BAVet122

### CODE REVIEW

File: new/PRS14615Maqoyi.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">መቆዪ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Maqoyi</persName>
+                    <persName><roleName type="title">ʾalaqā</roleName></persName>
+                    <floruit when="1913"/>
+                </person>
+                <listRelation>
+                    <relation name="saws:isCopierOf" active="PRS14615Maqoyi" passive="BAVet122">
+                        <desc>Gabra Māryām is mentioned as the scribe in the colophon of <ref type="mss" corresp="BAVet122"/> on f. 149v.</desc>
```
Comment (thea-m) [inline new/PRS14615Maqoyi.xml:55]: Maqoyi

**Review by thea-m — CHANGES_REQUESTED** [review]
Great, thank you!

**Review by thea-m — APPROVED** [review]

**Comment by thea-m** [comment]
@MassimoVilla do you prefer to merge your PRs yourself after adequate approval or do you prefer the reviewers to merge?

**Comment by MassimoVilla** [comment]
> @MassimoVilla do you prefer to merge your PRs yourself after adequate approval or do you prefer the reviewers to merge?

Both solutions are fine. For persons and works, I can also merge my PRs myself. For bigger PRs (like MSS), I would prefer someone else, just to be sure there are no further requests for change.

**Pull request merged**